### DISCUSSION TITLE
error when install eslint plugin

### DISCUSSION THREAD

**Discussion by mfaridzia** [eslint/eslint#14071] [Q&A] [answered]
Hello, I have a problem every time I install some eslint-plugin, an error occurs and it fails to install, a few months ago it still works.

error:
npm i -D eslint-plugin-sort-destructure-keys (or another eslint plugin)
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@ndk%2fenv - Not found
npm ERR! 404 
npm ERR! 404  '@ndk/env@0.0.3' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 It was specified as a dependency of 'eslintcc'
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/lenov/.npm/_logs/2021-02-05T09_14_56_820Z-debug.log

**Comment by mdjermanovic** [comment, marked as answer]
Hi @mfaridzia!

This doesn't look like a problem with ESLint itself. 

> npm ERR! 404 '@ndk/env@0.0.3' is not in the npm registry.
> npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
> npm ERR! 404 It was specified as a dependency of 'eslintcc'

It seems one of the packages you're using (`eslintcc`) has dependency on package `@ndk/env@0.0.3` that doesn't exist or is private,  so it can't be installed. This is probably fixed by https://github.com/eslintcc/eslintcc/commit/7ec4a5a33574fab78594fe36b017c022b6a0685e so you could try with the latest version of `eslintcc`.

**Reply by mfaridzia** [reply]
Ah oke, thank you @mdjermanovic
